lib/markbind/lib/parser.js

```diff
@@ -34,7 +35,16 @@ const BOILERPLATE_FOLDER_NAME = '_markbind/boilerplates';
  * Utils
  */
 
+/**
+* @throws Will throw an error if a non-absolute path or path outside the root is given
+*/
 function calculateNewBaseUrl(filePath, root, lookUp) {
+  if (!path.isAbsolute(filePath)) {
+    throw new Error(`Non-absolute path given to calculateNewBaseUrl: '${filePath}'`);
+  }
+  if (!pathIsInside(filePath, root)) {
+    throw new Error(`File '${filePath}' given is not in root '${root}'`);
```

Comment:
"Path given '${filePath}' is not in root '${root}'" (consistent phrasing for "path given").